jsonnet/telemeter/prometheus/kubernetes/kubernetes.libsonnet

```diff
@@ -20,6 +20,8 @@ local k = import 'ksonnet/ksonnet.beta.3/k.libsonnet';
       rules: { groups: [] },
       htpasswdAuth: '',
       sessionSecret: '',
+      sar: '{"resource": "namespaces", "verb": "get", "resourceName": "telemeter-production", "namespace": "telemeter-production"}',
```

Comment:
this seems like a bit too much configurability here. I think we can optimize for our actual expected use case :), since we know how it is intended to be deployed. Anyone wild enough to use and want more configurability power this can import the jsonnet and use additional mixins to modify the generated manifests

Reply:
ack, see https://github.com/openshift/telemeter/pull/74#discussion_r230011249